README.md

```diff
@@ -58,6 +58,7 @@ Some old update logs are available at [Key Events](https://github.com/microsoft/
 
 External (Unofficial) Repositories
 ----------------------------------
+FLAML (AutoML library for efficient hyperparameter optimization): https://github.com/microsoft/FLAML
```

Comment:
I don't think that "efficient" word in this sentence adds any value, but just enlarges the description which should be compact. I don't think there are "inefficient" libraries for hyperparameter optimization 😄 .
```suggestion

FLAML (AutoML library for hyperparameter optimization): https://github.com/microsoft/FLAML
```

Reply:
sure. revised! thanks :)